test/src/config/index.js

```diff
@@ -6,7 +6,6 @@ dotenv.config();
 
 //switch to local if running locally
 const env = process.env.NODE_ENV;
-//const env = 'local'
```

Comment:
I think we should leave this in here. I like this method so we don't need to remember to set local on our machines. 
const env = process.env.NODE_ENV || 'local';